srcpkgs/fntsample/files/LICENSE.unicode

```diff
@@ -0,0 +1,33 @@
+COPYRIGHT AND PERMISSION NOTICE
```

Comment:
Where does this license come from?

Reply:
Not present in the official tarball?

Reply:
It’s from the bottom of the [full terms of use page](http://www.unicode.org/terms_of_use.html), which is what `Blocks.txt` file links to at line 4.

`Blocks.txt` is not in the package, but only referred to in the [readme](https://github.com/eugmes/fntsample#building):

> Additionally Unicode [blocks](https://unicode.org/Public/UNIDATA/Blocks.txt) file is required.

which links to a rolling release, but there is a versioned equivalent which I use in the distfiles.

Reply:
ok, I see.
I think it is possible to do something like:
````
distfiles="https://github.com/eugmes/fntsample/archive/release/${version}.tar.gz
 https://unicode.org/Public/11.0.0/ucd/Blocks.txt
 http://www.unicode.org/copyright.html"
skip_extraction="copyright.html"
checksum="f5d6da0ea61235b050e3b3334349c2545786e18da23d4ef9ff0a0b4ca6e7f0cb
 0b457b66c788a97c8521e265f0b793d4ed911356d39eb61029f9cef8554cd052
 341d64a621d779d29ed33b771798a0bc5c2acf88f28a00f355d0f70c14fe8a06"

post_install() {
	vlicense ${XBPS_SRCDISTDIR}/${pkgname}-${version}/copyright.html
}
````

Reply:
Since `Blocks.txt` refers to `terms_of_use.html`, I think that should be used instead, and it redirects to `copyright.html` anyway, for the time being.

Also we would get a shasum mismatch when the license is updated, but I assume that would be fixed manually on demand for each bump?

```sh
distfiles="https://github.com/eugmes/fntsample/archive/release/${version}.tar.gz
 https://unicode.org/Public/11.0.0/ucd/Blocks.txt
 https://unicode.org/terms_of_use.html"
skip_extraction="terms_of_use.html"
checksum="f5d6da0ea61235b050e3b3334349c2545786e18da23d4ef9ff0a0b4ca6e7f0cb
 0b457b66c788a97c8521e265f0b793d4ed911356d39eb61029f9cef8554cd052
 341d64a621d779d29ed33b771798a0bc5c2acf88f28a00f355d0f70c14fe8a06"

post_install() {
	vlicense ${XBPS_SRCDISTDIR}/${pkgname}-${version}/terms_of_use.html
}
```